doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -114,6 +114,8 @@ Orion always provides datetime attributes/metadata using the format `YYYY-MM-DDT
 timezone (which is the best default option, as clients/receivers may be running in any timezone). This may change in the
```

Comment:
I think the change corresponding to this PR deserves a CNR entry. At the end, it involves a functional change to improve integration with legacy system, so I think is a good idea to mention it.